src/Api/Enum/CryptographicAlgorithmEnum.php

```diff
@@ -2,20 +2,32 @@
 
 namespace Azimo\Apple\Api\Enum;
 
+/**
+ * appleid 支持的 auth keys
```

Comment:
I'm fine with adding PHPDoc. However, the whole library supports only English (even though it's not even my native language), so please use English in comments only.